scripts/test_and_docs.py

```diff
@@ -0,0 +1,52 @@
+import os
+import sys
+
+import sphinx.ext.apidoc as apidoc
+import sphinx.cmd.build as build
+import python_on_whales
+import pytest
+import dotenv
+
+import docs.source.utils.patching as patching
+from .codestyle import lint
+from .clean import clean_docs
+
+
+def test():
+    test_coverage_threshold = 95
+    test_coverage_report = "html"
+    test_coverage_term = "term"
+    test_coverage = "dff"
+
+    dotenv.load_dotenv(".env_file")
+    pytest.main(
+        [
+            f"--cov-fail-under={test_coverage_threshold}",
+            "--cov-report",
+            test_coverage_report,
+            "--cov-report",
+            test_coverage_term,
+            f"--cov={test_coverage}",
+            "tests/",
+        ]
+    )
+
+
+def test_all():
+    if "linux" in sys.platform:
+        docker = python_on_whales.DockerClient(compose_files=["docker-compose.yml"])
+        docker.compose.up(detach=True, wait=True)
+    test()
+    if "linux" in sys.platform:
+        docker.compose.down(remove_orphans=False)
+    lint()
+
+
+def docs():
```

Comment:
Could you implement patching as suggested [here](https://github.com/deeppavlov/dialog_flow_framework/pull/91#discussion_r1141479772) and see if it works?